src/dxgi/dxgi_monitor.cpp

```diff
@@ -142,6 +145,26 @@ namespace dxvk {
                                  / pMode->RefreshRate.Denominator;
     }
     
+    /* Compare new mode with current and change if necessary */
+    if (EnumDisplaySettingsW(monInfo.szDevice, ENUM_CURRENT_SETTINGS, &currentMode)) {
+      if (currentMode.dmPelsWidth == devMode.dmPelsWidth &&
+        currentMode.dmPelsHeight == devMode.dmPelsHeight &&
+        currentMode.dmBitsPerPel == devMode.dmBitsPerPel &&
+        (currentMode.dmDisplayFrequency == devMode.dmDisplayFrequency ||
+        !(devMode.dmFields & DM_DISPLAYFREQUENCY)) &&
+        (currentMode.dmDisplayFlags == devMode.dmDisplayFlags ||
+        !(devMode.dmFields & DM_DISPLAYFLAGS)))
+      {
+        Logger::info(str::format("DXGI: Skipping redundant mode setting call: ",
```

Comment:
I highly doubt we need a log message to inform the user that nothing happened.

Reply:
Agree, but if this will have some problems, log message will help with debugging. If it's too much, I will remove